intelmq/bots/outputs/smtp_batch/output.py

```diff
@@ -290,7 +292,7 @@ def prepare_mails(self):
                     try:
                         zf.writestr(filename + ".csv", output.getvalue())
                     except Exception:
-                        self.logger.error(f"Cannot zip mail {mail_record}")
+                        self.logger.error(f"Cannot zip mail %s", mail_record)
```

Comment:
remove the `f` modifier as above and format the email address (adds parenthesis automatically)

```suggestion
                        self.logger.error("Cannot zip mail %r", mail_record)
```

Reply:
Thanks, fixed!